src/main/java/com/kodilla/ecommercee/mapper/OrderMapper.java

```diff
@@ -0,0 +1,117 @@
+package com.kodilla.ecommercee.mapper;
+
+import com.kodilla.ecommercee.dao.CartDao;
+import com.kodilla.ecommercee.dao.UserDao;
+import com.kodilla.ecommercee.domain.Cart;
+import com.kodilla.ecommercee.domain.Order;
+import com.kodilla.ecommercee.domain.User;
+import com.kodilla.ecommercee.domain.dto.OrderDto;
+import org.springframework.stereotype.Component;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+@Component
+public class OrderMapper {
+
+    CartDao cartDao;
```

Comment:
@Autowired

Reply:
dlaczego tu jest CartDao i UserDao a nie CartDto i UserDto?

Reply:
> @Autowired

NIE  https://www.baeldung.com/constructor-injection-in-spring

Reply:
CartDto i UserDto to nie sa beany